src/core/main.cpp

```diff
@@ -120,36 +120,36 @@ void printHelp()
 {
 	printf( "LMMS %s\n"
 		"Copyright (c) 2004-%s LMMS developers.\n\n"
-		"usage: lmms [ -r <project file> ] [ options ]\n"
+		"Usage: lmms [ -r <project file> ] [ options ]\n"
```

Comment:
@M374LX it tends to be uppercase for me as well (Ubuntu) - tested with `rm`, `rsync`, `firefox`, `more`, `subl` (Sublime Text), `gedit`. `tput` is the only one I found that uses a lowercase convention.